src/backend/base/langflow/components/vectorstores/astradb.py

```diff
@@ -571,295 +597,327 @@ def _initialize_collection_options(self, api_endpoint: str | None = None):
-                build_config["api_endpoint"]["value"] = build_config["database_name"]["options_metadata"][
-                    index_of_name
-                ]["api_endpoint"]
+        # Reset collection list if database selection changes
+        if field_value not in build_config["database_name"]["options"]:
+            build_config["database_name"]["value"] = ""
+            return build_config
 
-                # Reset the provider options
-                build_config = self.reset_provider_options(build_config)
+        # Get the api endpoint for the selected database
+        index = build_config["database_name"]["options"].index(field_value)
+        build_config["api_endpoint"]["value"] = build_config["database_name"]["options_metadata"][index]["api_endpoint"]
 
-            # Reset the list of collections we have based on the token provided
-            return self.reset_collection_list(build_config)
+        # Get the org_id for the selected database
+        org_id = build_config["database_name"]["options_metadata"][index]["org_id"]
+        if not org_id:
+            return build_config
 
-        # Hide embedding model option if opriona_metadata provider is not null
-        if field_name == "collection_name" and not isinstance(field_value, dict):
-            # Assume we will be autodetecting the collection:
-            build_config["autodetect_collection"]["value"] = True
-
-            # Reload the collection list
-            build_config = self.reset_collection_list(build_config)
-
-            # Set the options for collection name to be the field value if its a new collection
-            if field_value and field_value not in build_config["collection_name"]["options"]:
-                # Add the new collection to the list of options
-                build_config["collection_name"]["options"].append(field_value)
-                build_config["collection_name"]["options_metadata"].append(
-                    {
-                        "records": 0,
-                        "provider": None,
-                        "icon": "vectorstores",
-                        "model": None,
-                    }
-                )
+        # Get the database id for the selected database
+        db_id = self.get_database_id_static(api_endpoint=build_config["api_endpoint"]["value"])
+        keyspace = self.get_keyspace() or "default_keyspace"
+
+        # Update the helper text for the embedding provider field
+        template = build_config["collection_name"]["dialog_inputs"]["fields"]["data"]["node"]["template"]
+        template["02_embedding_generation_provider"]["helper_text"] = (
+            "To create collections with more embedding provider options, go to "
+            f'<a class="underline" target="_blank" rel="noopener noreferrer" '
+            f'href="https://astra.datastax.com/org/{org_id}/database/{db_id}/data-explorer?createCollection=1&namespace={keyspace}">'
```

Comment:
is there a way we can create this link on the UI? Using `dangerouslySetInnerHTML` could open us up to XSS vulnerabilities

Reply:
another option could be markdown here